client/src/RoutingContainer.tsx

```diff
@@ -1,21 +1,22 @@
-import React, { FC } from 'react';
-import { BrowserRouter as Router, Route } from 'react-router-dom';
-import ScrollToTop from './components/ScrollToTop';
-import CanvasPageContainer from './canvas/containers/CanvasPageContainer';
-import AboutPage from './about/components/About';
-import Header from './header/containers/HeaderContainer';
-import Footer from './footer/components/Footer';
+import React, { FC } from "react";
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import ScrollToTop from "./components/ScrollToTop";
+import CanvasPageContainer from "./canvas/containers/CanvasPageContainer";
+import AboutPage from "./about/components/About";
+import Header from "./header/containers/HeaderContainer";
+import Footer from "./footer/components/Footer";
+import ColorPicker from "./colorPicker/colorPicker";
```

Comment:
Looks like this line is the only new one.

Reply:
Looks like you're converting all single quotes to double quotes... is there a reason for that?

Reply:
I use the Prettier extension which sets a rule for frontend and it must enforce single quotes. Two ways to fix this: I can change the preference, or we all download Prettier so that we no longer have this problem as the style will be standardized

Reply:
In general Prettier is great - could you just change that setting to use single quotes instead of double quotes? That's the typical convention